### DISCUSSION TITLE
How do you filter by partial match in array of strings in Prisma?

### DISCUSSION THREAD

**Discussion by dev-tashes** [prisma/prisma#13919] [Q&A] [answered]
Say I have the following Prisma schema:

```
model Article {
    id String @db.ObjectID
    tags String[]
}
```

There are articles like this in the database:

```
id: 1, tags: [ 'tag1', 'tag2', 'tag3' ]
id: 2, tags: [ 'taga', 'tag1' ]
id: 3, tags: [ 'water', 'tag3' ]
```

How could I make an `article.findMany` filter where a string could be partially matched against all the tags?

So for example, if my search string is `ag1` it will return ids `1` and `2`, or if the search string is `at`, it would return id `3`.

**Comment by austincrim** [comment, marked as answer]
Hey @dev-tashes 👋,

This is not possible in Prisma. You would need to do a raw query (probably [`findRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#findraw) if you're using Mongo) to accomplish this.

**Reply by dev-tashes** [reply]
Alright thanks for letting me know!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
